apps/designer/env-check.js

```diff
@@ -13,7 +13,7 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 require("dotenv").config();
 
-const REQUIRED_ENVS = ["DATABASE_URL", "AUTH_SECRET"];
+const REQUIRED_ENVS = ["DATABASE_URL", "AUTH_SECRET", "DESIGNER_DOMAIN"];
```

Comment:
does this mean we need to set it locally too? Should that be required for production only?

Reply:
Yes, you need to set it locally as well, as I removed the hardcoded domain from the codebase.